pom.xml

```diff
@@ -235,6 +235,7 @@
                      <exclude>parquet/org/**</exclude>
                      <exclude>parquet/column/**</exclude>
                      <exclude>parquet/hadoop/ParquetInputSplit</exclude>
+                     <exclude>parquet/thrift/struct/**</exclude>
```

Comment:
@julienledem thoughts on this? It's easy to keep the old constructor and mark it deprecated, but it really shouldn't be used, there's no reasonable default for isUnion other than figuring out if it really is a union.